example.env

```diff
@@ -6,5 +6,6 @@
 # SERVER ENV
 PORT=''
```

Comment:
completely optional- but you could add a server port (for your server index file) and db port both into you env - might be good practice but isn't necessary if your localhost port is hardcoded into your server setup

Reply:
Agreed. I have the server port in my own .env file. This is just for the example file so the user has the option to change the port, but if they don't provide a port, then it is default to 3000.